src/sidebar/services/annotations-exporter.tsx

```diff
@@ -189,18 +189,30 @@ export class AnnotationsExporter {
               </a>
             </p>
 
-            <dl>
-              <dt>Group</dt>
-              <dd>{groupName}</dd>
-              <dt>Total users</dt>
-              <dd>{uniqueUsers.length}</dd>
-              <dt>Users</dt>
-              <dd>{uniqueUsers.join(', ')}</dd>
-              <dt>Total annotations</dt>
-              <dd>{annotations.length}</dd>
-              <dt>Total replies</dt>
-              <dd>{replies.length}</dd>
-            </dl>
+            <table>
+              <tbody>
+                <tr>
+                  <td>Group:</td>
```

Comment:
I was planning to use `th` for the first column, but by default they are center-aligned, and I didn't want to add inline styles